sdk/communication/Azure.Communication.Sms/src/Models/SendMessageRequest.cs

```diff
@@ -3,8 +3,10 @@
 
 using Azure.Core;
 
-namespace Azure.Communication.Sms
+namespace Azure.Communication.Sms.Models
 {
+    /// <summary> Send Message Request. </summary>
```

Comment:
is not the summary automatically added to the generated code based on the swagger?